static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -113,8 +114,14 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
     const isCurrentUser = currentUser.email === email;
     const showRemoveButton = !isCurrentUser;
     const showLeaveButton = isCurrentUser;
+    const isInvite = inviterName !== null;
+    const canRemoveInvite =
```

Comment:
Should this include isInvite?

Reply:
No, we only use `isInvite` to determine what disabled message to show the user (line 214). In `canRemoveInvite`, we already check if this is an invite through `inviterName === currentUser.name`.